src/SDDPoptimize.jl

```diff
@@ -208,42 +209,100 @@ function optimize(model::SPModel,
 
     stopping_test::Bool = false
     iteration_count::Int64 = 0
-    n::Int64 = param.forwardPassNumber
-
-
 
     while (iteration_count < param.maxItNumber) & (~stopping_test)
+        # Time execution of current pass:
+        tic()
+
         # Build given number of scenarios according to distribution
         # law specified in model.noises:
         aleas = simulate_scenarios(model.noises ,
                                     (model.stageNumber,
                                      param.forwardPassNumber,
                                      model.dimNoises))
+
+        # Forward pass
         costs, stockTrajectories, _ = forward_simulations(model,
                             param,
                             V,
                             problems,
-                            n,
                             aleas)
 
-        V0 = backward_pass(model,
+        # Backward pass
+        V0 = backward_pass!(model,
                       param,
                       V,
                       problems,
                       stockTrajectories,
                       model.noises)
 
-        iteration_count+=1;
-
+        iteration_count += 1
         upb = upper_bound(costs)
+        time = toq()
+
         if display
             println("Pass number ", iteration_count,
-                    "  Upper bound: ", upb,
-                    "  V0: ", V0)
+                    "\tEstimation of upper-bound: ", upb,
+                    "\tV0: ", V0,
+                    "\tTime: ", time)
         end
 
-        stopping_test = test_stopping_criterion(V0, upb, param.sensibility)
+    end
+
+    if display
+        println("Estimate upper-bound with Monte-Carlo ...")
+        upb = estimate_upper_bound(model, param, V, problems)
+        println("Estimation of upper-bound: ", upb,
```

Comment:
I feel that we should rework the presentation a bit. I would like to have the following display : 
- estimation of cost of the solution (fiability 95%) : mean(costs) +/- 1.96 stdev(costs)/sqrt(size(costs))
- exact lower bound : V0
-  "Gap < (V0-upb)/V0) with prob. > 97.5\%"